cookbooks/cdo-users/templates/default/bashrc.erb

```diff
@@ -57,9 +57,9 @@ if [ -n "$force_color_prompt" ]; then
 fi
 
 if [ "$color_prompt" = yes ]; then
-    PS1='${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@<%= node.name %>\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
+PS1='${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
 else
-    PS1='${debian_chroot:+($debian_chroot)}\u@<%= node.name %>:\w\$ '
+PS1='${debian_chroot:+($debian_chroot)}\u@\h:\w\$ '
```

Comment:
Forgot to mention two more details about this:

1. This change actually reverts this template back to the [original](http://bazaar.launchpad.net/~doko/+junk/pkg-bash-debian/view/head:/skel.bashrc#L60), so in terms of maintainability this will be an improvement. (The only other customizations we've done to this file are the environment variables at the bottom, so soon we might be able to remove this copy/pasted template entirely.)

2. This change is needed for the upcoming immutable server process: since we'll no longer be doing a full Chef Client run on new frontends (just changing the hostname to match the instance ID), before this change we would need to run the `cdo-users` in order to regenerate this template to update the Bash prompt. With this change, once we update the hostname, the Bash prompt will update on its own.